Examples/Zika/zika-virus-intervention-logic.xml

```diff
@@ -58,483 +58,625 @@
+                     <operand localId="67" valueType="t:String" value="final" xsi:type="Literal"/>
                   </operand>
-                  <operand xsi:type="Equal">
-                     <operand path="reference" xsi:type="Property">
-                        <source path="encounter" scope="O" xsi:type="Property"/>
+                  <operand localId="74" xsi:type="Equal">
+                     <operand localId="71" path="reference" xsi:type="Property">
+                        <source localId="70" path="encounter" scope="O" xsi:type="Property"/>
                      </operand>
                      <operand toType="fhir:string" xsi:type="Convert">
-                        <operand path="id" xsi:type="Property">
-                           <source name="Current Encounter" xsi:type="ParameterRef"/>
+                        <operand localId="73" path="id" xsi:type="Property">
+                           <source localId="72" name="Current Encounter" xsi:type="ParameterRef"/>
                         </operand>
                         <toTypeSpecifier name="fhir:string" xsi:type="NamedTypeSpecifier"/>
                      </operand>
                   </operand>
                </where>
-               <sort>
-                  <by direction="desc" xsi:type="ByExpression">
-                     <expression path="effectiveDateTime" scope="O" xsi:type="Property"/>
+               <sort localId="79">
+                  <by localId="78" direction="desc" xsi:type="ByExpression">
+                     <expression localId="77" path="effectiveDateTime" scope="O" xsi:type="Property"/>
                   </by>
                </sort>
             </source>
          </expression>
       </def>
-      <def name="Has Zika Exposure Assessment" context="Patient" accessLevel="Public">
-         <expression xsi:type="Not">
+      <def localId="85" name="Has Zika Exposure Assessment" context="Patient" accessLevel="Public">
+         <annotation xsi:type="a:Annotation">
+            <a:s>define &quot;Has Zika Exposure Assessment&quot;:&#xd;
+  &quot;Zika Exposure Assessment Result&quot; is not null</a:s>
+         </annotation>
+         <expression localId="84" xsi:type="Not">
             <operand xsi:type="IsNull">
-               <operand name="Zika Exposure Assessment Result" xsi:type="ExpressionRef"/>
+               <operand localId="83" name="Zika Exposure Assessment Result" xsi:type="ExpressionRef"/>
             </operand>
          </expression>
       </def>
-      <def name="Zika Symptoms" context="Patient" accessLevel="Public">
-         <expression xsi:type="Query">
-            <source alias="C">
-               <expression dataType="fhir:Condition" codeProperty="code" xsi:type="Retrieve">
+      <def localId="105" name="Zika Symptoms" context="Patient" accessLevel="Public">
+         <annotation xsi:type="a:Annotation">
+            <a:s>define &quot;Zika Symptoms&quot;:&#xd;
```

Comment:
Note that these annotations are encoding the carriage return as `&#xd;`.  That's probably not optimal (and doesn't happen on Mac) -- but probably isn't harmful either, so I'll leave it as-is for now.